security/tor/Makefile

```diff
@@ -1,7 +1,7 @@
 PLUGIN_NAME=          tor
-PLUGIN_VERSION=       1.1
+PLUGIN_VERSION=       1.2
 PLUGIN_COMMENT=       The Onion Router
-PLUGIN_DEPENDS=       tor
+PLUGIN_DEPENDS=       tor ruby
```

Comment:
comment: I know it's fast dev for you but it would be neater without it ;)

Reply:
PHP is not really an alternative (worse readable) and shell will probably not work :(